crates/proof-of-sql/src/base/polynomial/lagrange_basis_evaluation_test.rs

```diff
@@ -108,46 +154,46 @@ fn compute_truncated_lagrange_basis_sum_gives_correct_values_with_3_variables()
 }
 
 #[test]
-fn compute_truncated_lagrange_basis_sum_gives_correct_values_with_3_variables_using_dalek_scalar() {
```

Comment:
Since the test was formerly a duplicate of the test above I just modified it so that we check that the rho eval is correct with 3 variables.